<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWidget</string>
  </property>
  <widget class="QLabel" name="labelConnexion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Connexion au serveur</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameConnexion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>541</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse IP du serveur</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIP">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Num√©ro de port du serveur</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPort">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQuitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnexion">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditEtat">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelEtat">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de la connexion</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelInfoClients">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Informations Client</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameInfoClients">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>541</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButtonNomOrdi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de l'ordinateur distant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNomUtilisateur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de l'utilisateur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OS de l'ordinateur</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNomOrdi">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNomUtilisateur">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOS">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonArchitecture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Architecture processeur</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditArchitecture">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>ClientWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
